content/docs/concepts/security/index.md

```diff
@@ -287,6 +287,10 @@ When a client calls the `datastore` service, it extracts the `test-team` identit
 and checks whether `test-team` is allowed to run `datastore` with the secure naming information.
```

Comment:
I cannot comment on previous line in place, here is the line I would suggest to change "Suppose the malicious user successfully hacked the discovery service or DNS to map the `datastore` service name to the forged server." ->
Suppose the malicious user successfully hijacked (through DNS spoofing, BGP/route hijacking, ARP spoofing, etc.) the traffic sent to the `datastore` and redirect it to the forged server.

Reply:
Done